pkg/server/sqlflowserver.go

```diff
@@ -82,6 +78,9 @@ func (s *Server) Run(req *pb.Request, stream pb.SQLFlow_RunServer) error {
 			res, err = encodeRow(s)
 		case string:
 			res, err = encodeMessage(s)
+		case sf.WorkflowJob:
+			job := r.(sf.WorkflowJob)
+			res = &pb.Response{Response: &pb.Response_Job{Job: &pb.Job{Id: job.JobID}}}
```

Comment:
If you also update the pysqlflow, please refer to this PR in the pysqlflow PR.

Reply:
Sure, there is a task in the TODO list #1066